upgrade_tests/helpers/variants.py

```diff
@@ -108,7 +112,13 @@ class VersionError(Exception):
         ['["lo", "en*v*", "usb*", "vnet*", "macvtap*", "_vdsmdummy_", '
          '"veth*", "docker*", "tap*", "qbr*", "qvb*", "qvo*", "qr-*", '
          '"qg-*", "vlinuxbr*", "vovsbr*", "mountpoints", "partitions", '
-         '"blockdevice*"]']*2,
+         '"blockdevice*"]'] +
+        ['["lo", "en*v*", "usb*", "vnet*", "macvtap*", "_vdsmdummy_", '
+         '"veth*", "docker*", "tap*", "qbr*", "qvb*", "qvo*", "qr-*", '
+         '"qg-*", "vlinuxbr*", "vovsbr*"]'],
+        ['']*5 + ['*****'], ['dog8code']*5 + ['*****'],
```

Comment:
we should not expose psswords like that. We should use GitLab config files variables here instead of directly hard passwd in the public repo.

Reply:
Fixed.